packages/fxa-auth-server/scripts/convert-customers-to-stripe-automatic-tax/convert-customers-to-stripe-automatic-tax.ts

```diff
@@ -44,6 +45,8 @@ export class StripeAutomaticTaxConverter {
 
     // TODO FXA-6581: Populate this with a JSON file
     this.ipAddressMap = {};
+
+    this.ineligibleProducts = ['prod_HEJ13uxjG4Rj6L', 'prod_HeWOjYtYcEjAzV'];
```

Comment:
Opted to hard code instead of adding a config parameter, since I believe this is a temporary script? Happy to add config params though.